added nlu examples

C-K-Loan opened the pull request:
# NLU examples added

https://github.com/JohnSnowLabs/nlu

# **What can NLU do in 1 line of Python code?**

- **350+** pre-trained models
- **100+** of the latest NLP word embeddings ( BERT, ELMO, ALBERT, XLNET, GLOVE, BIOBERT, ELECTRA, COVIDBERT) and different variations of them
- **50+** of the latest NLP sentence embeddings ( BERT, ELECTRA, USE) and different variations of them
- **50+** Classifiers
- **50+** Supported languages
- Named Entity Recognition (NER)
- Part of Speech(POS)
- Labeled and Unlabeled Dependency parsing
- Spell Checking
- Various Text Preprocessing and Cleaning algorithms

vkocaman commented:
@C-K-Loan Collab -->> Colab

- Sent Det. >> not a working example can be seen, and there are error messages in the output. We can also have an example for SDDL (DL version of SD). Turning on and off the DL version as a param.
-   NLU_Question_Classification_Example .ipynb.ipynb >> .ipynb.ipynb
- BIOBERT Word Embeddings with NLU >> sample text could be clinical (not sarcasm related)

C-K-Loan commented:
- Added comparision between SDLL and pragmatic SD. Also added comparison to Spacy and Multi Lingual SDLL examples
- Typos fixed
- Medical dataset for BIOBERT added
- Notebook showcasing training for classifier_dl and saving/loading a pipeline
- Notebook showcasing sentence similarity added
